src/main/database/driveFilesDB.ts

```diff
@@ -0,0 +1,126 @@
+import { DatabaseAdapter } from './adapters/base';
+import { RemoteSyncedFile } from '../remote-sync/helpers';
+import SQLite from 'better-sqlite3';
+
+export function generateCreateTableQuery(): string {
+  const query = `CREATE TABLE IF NOT EXISTS drive_files (
```

Comment:
Should we use Sequelize to create a mini-backend as a layer to make this easier to maintain?

Reply:
Take into count that all the migrations history will be easy to run and keep track of, so each time we can run the migrations if the app autoupdates.

Reply:
We can use TypeORM then, which seems to have support and is the alternative to Sequelize: 
https://www.npmjs.com/package/typeorm-with-better-sqlite3

Reply:
Repo here, as that package seems to be an adaptation: 
https://github.com/typeorm/typeorm#readme